azurerm/import_arm_image_test.go

```diff
@@ -0,0 +1,49 @@
+package azurerm
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccAzureRMImage_importStandalone(t *testing.T) {
+	ri := acctest.RandInt()
+	userName := "testadmin"
+	password := "Password1234!"
+	hostName := "tftestcustomimagesrc"
+	dnsName := fmt.Sprintf("%[1]s.westcentralus.cloudapp.azure.com", hostName)
+	sshPort := "22"
```

Comment:
Not sure if we could. I missed the random seed needed to avoid collisions between host names initially, but now that I put it in, the random seed is needed to generalize the VM since the DNS Name is one of the params to that function now. Open to different ideas here as well.